### PR TITLE
Add instructions to set up GMT mirrors

### LINKED ISSUE #21: Have instructions for potential GMT distribution and data server mirrors
Seems we should have instructions on our website for potential GMT distribution and data server mirrors.  There should one mirror script for an ftp server (to serve the GMT installers) [could just server the latest [< 1 Gb] or the entire history of releases [15 Gb], and one mirror script for a data server[50+ Gb].

_Originally posted by @PaulWessel in https://github.com/GenericMappingTools/gmt/issues/1675#issuecomment-536338053_

### PR DESCRIPTION
**Pull request by seisman**
This PR adds some instructions to set up a GMT FTP mirror and data server mirror. Feel free to make any changes if the sentences don't read well.

Closes #21.
Closes https://github.com/GenericMappingTools/gmtserver-admin/issues/80.

**Screenshot:**

image

**Preview**: https://website-git-mirror-instructions.gmt.vercel.app/mirrors/#becomming-a-gmt-mirror

### CODE REVIEW

File: mirrors/index.rst [hunk 1]
```diff
@@ -45,4 +45,26 @@ Name              Address
...
-                          :alt: Europea Mirror Status
\ No newline at end of file
+                          :alt: Europea Mirror Status
+
+
+Becomming a GMT mirror
+----------------------
+
+You can help out the GMT community by running a mirror of
+the GMT FTP site (~XX GB) and/or the GMT data server (~XX GB).
```
Comment (seisman) [inline mirrors/index.rst:55]: @PaulWessel Can you give the estimated size of the FTP server and data server?

**Comment by PaulWessel** [comment]
The SOEST ftp/gmt directory has 21 Gb but that also counts an early "daa" directory of 3.9 Gb with our first-generation earth_relief_*.grd files.  Given ubuntu users will access them until 2028 I guess we should say 21 Gb or ~20 Gb.

The data server (just the data directory) is currently at 57 Gb.

File: mirrors/index.rst (hunk 1 above)
Comment (seisman) [inline mirrors/index.rst:55]:
```suggestion
the GMT FTP site (~20 GB) and/or the GMT data server (~60 GB).
```

**Comment by seisman** [comment]
Ping @PaulWessel to review the PR.

**Review by PaulWessel — APPROVED** [review]

**Pull request merged**